tests/datastore_examples/mdp/era5_1000hPa_danra_100m_winds/config.yaml

```
@@ -0,0 +1,12 @@
+datastore:
+  kind: mdp
+  config_path: danra.datastore.yaml
+datastore_boundary:
+  kind: mdp
+  config_path: era5.datastore.yaml
```

Could we maybe do something like following instead:

```suggestion
boundary:
  datastore:
    kind: mdp
    config_path: era5.datastore.yaml
```

```suggestion
boundary:
  point_width: 30
```

And expand the config spec so that either `boundary.point_width` or `boundary.datastore` should be defined? Then we could keep the existing option where we force with the LAM domain data on the boundary. Just in cases where people haven't got boundary data or want to examine the effect of changing the two. Just an idea :)

---

If you want that setup you can do the splitting of the boundary from the interior first, then build two mdp-zarrs based on those and load in this setup. I don't think we need to preserve the option to load the boundary forcing from the same datastore here.

---

I think removing all the code related to these existing `interior_masks` is a good thing, as it makes the code easier to understand. Creating two zarrs seems straight-forward enough. And for those quickly wanting to test something they can:
- use the future hello_world example
- run an experiment with boundary=0, which should technically be possible, right @joeloskarsson

---

Yes, I have made the boundary forcing completely optional, so everything can be ran without it (tested with the DANRA example). This will be the behavior if no boundary datastore is given.